subsys/bluetooth/controller/hci/hci.c

```diff
@@ -21,6 +21,8 @@
 #include <zephyr/bluetooth/hci_vs.h>
 #include <zephyr/bluetooth/buf.h>
 
+#include "../common/hci_common_internal.h"
```

Comment:
```suggestion
#include "common/hci_common_internal.h"
```

`common` should be in the include path